code/acting/src/acting/coordinate_transformation.py

```diff
@@ -13,18 +13,18 @@
 
 # Class to choose a map with a predefined reference point
 class GeoRef(Enum):  # todo: add values for other towns
-    TOWN01 = 0, 0, 0
-    TOWN02 = 0, 0, 0
-    TOWN03 = 0, 0, 0
-    TOWN04 = 0, 0, 0
-    TOWN05 = 0, 0, 0
-    TOWN06 = 0, 0, 0
-    TOWN07 = 0, 0, 0
-    TOWN08 = 0, 0, 0
-    TOWN09 = 0, 0, 0
-    TOWN10 = 0, 0, 0
-    TOWN11 = 0, 0, 0
-    TOWN12 = 35.25000, -101.87500, 331.00000
+    TOWN01 = 0, 0, 0  # lat = 1.7e-08, lon = -5.6e-08, alt = 0.02
```

Comment:
Where do these values come from? They are all pretty much 0

Reply:
I tried to spawn at (0, 0, 0) on every map, and looked a the GPS sensor.
The only map where lat/lon was not equal to zero is town 12.